src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int INDEX_NAMES = 0;
+    private static final int INDEX_PRICE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            Files.deleteIfExists(Path.of(toFileName));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        File inputFile = new File(fromFileName);
+        File outputFile = new File(toFileName);
+        String[] array = getStringFile(inputFile).split("\r\n");
+        try {
+            outputFile.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+        writeToFile(getCountResult(array), outputFile);
+    }
+
+    private String getStringFile(File file) {
+        StringBuilder builder = new StringBuilder();
+        try {
+            builder.append(Files.readString(file.toPath()));
```

Comment:
You can try to use method [readAllLines](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-)